script/import_metadata

```diff
@@ -16,7 +16,7 @@ end
 
 
 puts "Loading environment..."
-require File.join(File.dirname(__FILE__), '../config/environment.rb')
+require_relative('../config/environment')
```

Comment:
I think the most flexible is something like this:
```ruby
File.join(File.expand_path(File.dirname(__FILE__)), '..', 'config', 'environment')
```
`File.join` will ensure you're using the correct directory separator, e.g. `/`.
`File.expand_path` will give you an absolute path.
`File.dirname(__FILE__)` will give you the directory name of the _current file_, so you'll get the right dir no matter where you are calling the script from.

It's stupidly verbose, each method does something different.